### PR TITLE
adding algolia search key

### PR DESCRIPTION
**Pull request by shaniR**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/theme/material/base.html [hunk 1]
```diff
@@ -221,13 +221,12 @@ <h2 id="__source">{{ lang.t("meta.source") }}</h2>
...
-      inputSelector:".md-search__input",
-      algoliaOptions: {"facetFilters":["language:en","version:3.2.0"]}
-      }
-  )</script>
+  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+  <script type="text/javascript"> docsearch({
+         apiKey: '1c63886380ded014e77714ad5fbc27d2',
+         indexName: 'wso2_mg',
+         inputSelector:".md-search__input"
+         algoliaOptions: {"facetFilters":["language:en","version:3.2.0"]}
```
Comment (tmkasun) [inline en/theme/material/base.html:229]: @praminda @shaniR 

We need to add meta tags in the respective pages in order to get facet based filtering

example: https://github.com/wso2/docs-apim/pull/857/files#diff-3e47efd75747b16701313fb6eeb54df3R14-R15

Comment (tmkasun) [reply en/theme/material/base.html:229]: ahh ok it's there 😄 
https://github.com/wso2/docs-apim/pull/857/files#diff-3e47efd75747b16701313fb6eeb54df3R14-R15

Comment (praminda) [reply en/theme/material/base.html:229]: We'll add this in one of the next PRs. Thanks for pointing it out

**Comment by praminda** [comment]
Search is still not working at https://mg.docs.wso2.com/en/latest/.
How can we find the reason for this? Do we need to wait sometime after adding the search Key?

**Pull request merged**